mod.ts

```diff
@@ -64,7 +64,7 @@ export class RhumRunner {
    *     Rhum.asserts.assertEquals(true, false); // fail
    */
   // deno-lint-ignore ban-types Reason for this is, deno lint no longer allows `Function` and instead needs us to be explicit: `() => void`, but  because  we couldn't use that to  type the properties (we would just be copying Deno's interfaces word for word), we have to deal with `Function
-  public asserts: { [key: string]: Function } = asserts;
+  public asserts: { [key in assertions]: Function } = asserts;
```

Comment:
Problem with this is that we don't have function support when you autocomplete.
If you use `public asserts = asserts` we have that support, but it doesn't build because of a type problem

Reply:
What do you mean by “autocomplete”? Like using Rhum.asserts. Or the asserts Parma’s?

Reply:
Essentially with this PR, we will have autocomplete in terms of the methods name that appears, but not their respective arguments.
- With `public asserts: { [key in assertions]: Function } = asserts;`, when typing `Rhum.asserts.*` it autocompletes to `Rhum.asserts.assertEquals`.
- With `public asserts = asserts`, when typing `Rhum.asserts.*` it autocompletes to `Rhum.asserts.assertEquals(first, second)` (don't actually know the param names here).

Reply:
we could just do this then:
```
const asserts: {
  assertEquals: StdAsserts.assertEquals,
  ...
} = 
  ...
}
```
? and say a new one is developed by the deno team, if a user tries to use it (which they won't be able to), then can just make an issue - i think the main point is we would already have a tonne of assertion methods

Reply:
But do you understand the point I'm trying to make? Other than that, everything is okay by me. Actually, it already was before :D

Reply:
Yeah i complete get you, essentially we have 2 problems:
* Add autocomplete to `Rhum.asserts` methods (this pr solved)
* Add autocomplete to the method parameters (this pr doesn't solve but you raised)

Whilst yeah, this PR only has a positive change, you did raise the point of the parameters which is **ideal** to have of course

What do you think about doing `const asserts: { the asserts } = { ... }`, so we can have autocomplete for methods **and** parameters?

Reply:
I tried playing with that a bit, but I wasn't able to make the tests pass. Can you try (using this branch as well) and I check out if it works as well on mine (I'm using VIM, but with a compatibility layer that uses VSCode extension to give autocomplete).

Reply:
Yeah i'll give it try 👍

Reply:
Unable to get it working... i think for now the autocompletion of methods is just a bonus, and i'll make an issue to track adding intellisense for the param stuff?